src/Core/src/Primitives/WebProcessTerminated.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Microsoft.Maui
+{
+	public class WebProcessTerminated
```

Comment:
Not sure if we should add the `EventArgs` suffix to match `PlatformPointerEventArgs`:


```suggestion
	public class WebProcessTerminatedEventArgs
```